discovery-server/Dockerfile

```diff
@@ -21,14 +21,14 @@ COPY ./role-service/pom.xml ./role-service/
 COPY ./site-service/pom.xml ./site-service/
 
 # cache most dependencies as a layer to be used in other builds while POMs don't change
-RUN mvn dependency:go-offline --batch-mode -q --projects ${SVC}
+RUN mvn dependency:go-offline --batch-mode -q -Dmaven.wagon.http.retryHandler.count=3 --projects ${SVC}
 
 # copy current service source
 COPY ./${SVC}/src ${SVC}/src
 
 RUN mvn package --projects ${SVC}
 
-FROM openjdk:11.0.9.1-jre-buster
+FROM adoptopenjdk/openjdk11:jre-11.0.10_9-debianslim
```

Comment:
Can we set the JDK (and other image) tags in some base config for Docker, so they don't have to be repeated in different files?

Reply:
I don't think there's a concept of parent docker files. We have talked about putting in just 1 file in the root folder to build it all, and I've tried it once. I will try to bring that change too in the next few days.